### PR TITLE
107 ods lib component top app bars

### PR DESCRIPTION
**Pull request by Tayebsed93**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Types of change

- Bug fix (non-breaking which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Refactoring (non-breaking change)
- Breaking change (fix or feature that would change existing functionality)

#### Accessibility

- [x] My change follows accessibility good practices

#### Design

- [x] My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
- [x] My change is compatible with responsive display

#### Documentation

- [X] My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- [X] Manually test on simulators and on a physical device (Android & iOS, orientation change, dark/light mode)
- [X] Code review
- [X] Design review
- [X] A11y review

### CODE REVIEW

File: app/CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Review the string in the app to use localization keys ([#129](https://github.com/Orange-OpenSource/ods-flutter/issues/129))
 - Add `OdsVerticalImageFirstCard` widget demonstration ([#8](https://github.com/Orange-OpenSource/ods-flutter/issues/8))
 - Add recipes Json file and use it ([#123](https://github.com/Orange-OpenSource/ods-flutter/issues/123))
-- Add customization bottom sheet to configure components ([#148](https://github.com/Orange-OpenSource/ods-flutter/issues/148)))
+- Add customization bottom sheet to configure components ([#148](https://github.com/Orange-OpenSource/ods-flutter/issues/148))
+- Add top app bar components ([#107](https://github.com/Orange-OpenSource/ods-flutter/issues/107)))
```
Comment (ludovic35) [inline app/CHANGELOG.md:19]: remove last )

File: NOTICE.txt [hunk 2]
```diff
@@ -24,18 +24,22 @@ ods-flutter/app/assets/1.5x/il_about.png
...
 ods-flutter/app/assets/3.0x/il_about.png
 ods-flutter/app/assets/3.0x/il_switches.png
 ods-flutter/app/assets/3.0x/il_typography.png
 ods-flutter/app/assets/3.0x/placeholder.png
+ods-flutter/app/assets/3.0x/il_app_bars_top_generic.png
 ods-flutter/app/assets/4.0x/il_about.png
 ods-flutter/app/assets/4.0x/il_switches.png
 ods-flutter/app/assets/4.0x/il_typography.png
 ods-flutter/app/assets/4.0x/placeholder.png
+ods-flutter/app/assets/4.0x/il_app_bars_top_generic.png
```
Comment (ludovic35) [inline NOTICE.txt:42]: Update Notice.txt file adding this asset in list

File: app/lib/ui/components/app_bars/top/component_top_app_bars.dart [hunk 3]
```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
```
Comment (ludovic35) [inline app/lib/ui/components/app_bars/top/component_top_app_bars.dart:1]: remove component_ in file name

File: app/lib/ui/components/utilities/customization_counter.dart [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    return SafeArea(
+      child: Padding(
+        padding: EdgeInsets.symmetric(horizontal: 16, vertical: 10),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Expanded(
+              child: ExcludeSemantics(
+                child: Text(
+                  'Action Count',
```
Comment (ludovic35) [inline app/lib/ui/components/utilities/customization_counter.dart:46]: should be localized

File: app/lib/ui/components/utilities/customization_counter.dart [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+            ),
+            Row(
+              children: [
+                MergeSemantics(
+                  child: Row(
+                    children: [
+                      Obx(
+                        () => Semantics(
+                          value:
+                              '${controller.count.toString()} actions in the top app bar, click to add another one',
```
Comment (ludovic35) [inline app/lib/ui/components/utilities/customization_counter.dart:59]: Should be localized

File: app/lib/ui/components/utilities/customization_counter.dart [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+                        style: Theme.of(context).textTheme.titleMedium,
+                      )),
+                ),
+                MergeSemantics(
+                  child: Row(
+                    children: [
+                      Obx(
+                        () => Semantics(
+                          value:
+                              '${controller.count.toString()} action in the top app bar, click to add one',
```
Comment (ludovic35) [inline app/lib/ui/components/utilities/customization_counter.dart:81]: same should be localized

**Review by ludovic35 — CHANGES_REQUESTED** [review]

File: app/lib/ui/components/utilities/customization_counter.dart (hunk 4 above)
Comment (Tayebsed93) [reply app/lib/ui/components/utilities/customization_counter.dart:46]: I will redo what you have done **component_count_row.dart**
We need to merge first : [23 ods component navigation bar](https://github.com/Orange-OpenSource/ods-flutter/pull/166/files#top)

File: app/lib/ui/components/utilities/customization_counter.dart (hunk 5 above)
Comment (Tayebsed93) [reply app/lib/ui/components/utilities/customization_counter.dart:59]: I will redo what you have done component_count_row.dart
We need to merge first : [23 ods component navigation bar](https://github.com/Orange-OpenSource/ods-flutter/pull/166/files#top)

File: app/lib/ui/components/utilities/customization_counter.dart (hunk 6 above)
Comment (Tayebsed93) [reply app/lib/ui/components/utilities/customization_counter.dart:81]: I will redo what you have done component_count_row.dart
We need to merge first : [23 ods component navigation bar](https://github.com/Orange-OpenSource/ods-flutter/pull/166/files#top)

File: NOTICE.txt (hunk 2 above)
Comment (Tayebsed93) [reply NOTICE.txt:42]: sorry can you specify what is expected?

Comment (ludovic35) [reply NOTICE.txt:42]: I think this line is missing:
ods-flutter/app/assets/il_app_bars_top_generic.png

Comment (Tayebsed93) [reply NOTICE.txt:43]: ok thank you

File: app/lib/ui/components/utilities/customization_counter.dart (hunk 6 above)
Comment (Tayebsed93) [reply app/lib/ui/components/utilities/customization_counter.dart:81]: it's ok i have retrieved **component_count_row.dart**

File: app/lib/ui/components/utilities/customization_counter.dart (hunk 5 above)
Comment (Tayebsed93) [reply app/lib/ui/components/utilities/customization_counter.dart:59]: it's ok i have retrieved **component_count_row.dart**

**Review by ludovic35 — APPROVED** [review]

**Pull request merged**